src/devices/Mcp3xxx/README.md

```diff
@@ -0,0 +1,25 @@
+# MCP3000, MCP3200 and MCP3300 ranges of Analog to Digital Converters
```

Comment:
please rename this to something like:

`MCP3xxx family of Analog to Digital Converters`

(this title is picked by the script which generates device listing (src/devices/README.md) so I'm more picky about this)